src/LegacyPortal.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import ReactDOM, { findDOMNode } from 'react-dom';
+import PropTypes from 'prop-types';
+
+const KEYCODES = {
+  ESCAPE: 27
+};
+
+const canUseDOM = !!(
+  typeof window !== 'undefined' &&
+  window.document &&
+  window.document.createElement
+);
```

Comment:
It looks like this is duplicated from Portal. It seems like it should be moved to a module that both can share, or outsourced to a dependency that multiple packages can share.